### DISCUSSION TITLE
Zitadel Is running slow

### DISCUSSION THREAD

**Discussion by Bhautik0110** [zitadel/zitadel#5104] [Q&A]
### Problem
I have deployed zitadel in kubernetes using Helm chart. But It running slowly. Login with password takes around ~7s. At the moment I have only a single user.

**Resource details**
```
Limits:   
 cpu:     250m               
 memory:  250Mi         
Requests:  
 cpu:      150m 
 memory:   150Mi 
```

Where I login, resource usage is near around:
```
cpu: 4m
memory: 125Mi
```

![image](https://user-images.githubusercontent.com/76986554/214557613-d3e8a272-644f-4807-94c4-0a77da0444cf.png)

Thanks.

**Comment by fforootd** [comment]
Hi There

Yes running a CPU Limit of 250mi is really low for bcrypts password hashing.

So you have two options.

1) Increase the CPU Limit to > 2000mi (we recommend doing this)
2) Reduce the bcrypt iteration through config https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml#L294 (this only applies to new and changed passwords)

Because of the bursty nature of password hashing you oftentimes will not see it properly in monitoring systems
